### PR TITLE
HTTP support

### PR DESCRIPTION
**Pull request by ThumNet**

### CODE REVIEW

File: src/RenderedHelper.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
 export class RenderedHelper {
-
+    
     public static show(options: { filename: string; content: string; }): void {
         if (!options.content) { return; }
 
         const firstChar = options.content[0];
         let extension = '.txt';
-        if (options.content.startsWith('<!DOCTYPE html>') || options.content.startsWith('<html>')) { extension = '.html'; }
+        if (options.filename.endsWith(CONSTANTS.httpFileExtension)) { extension = ''; }
```
Comment (LennartKoot) [inline src/RenderedHelper.ts:13]: Don't you want the extension to be `.http` here?

File: src/RenderedHelper.ts [hunk 2]
```diff
@@ -23,6 +24,17 @@ export class RenderedHelper {
         this._updateRenderedPdf(response);
     }
 
+    public static createHttpRequestContent(httpConfig: HttpStepConfiguration): string {
+        let headers = '';
+        if (httpConfig.headers) {
+            headers = Object.keys(httpConfig.headers).map(key => `${key}: ${httpConfig.headers?.[key]}`).join('\r\n');
```
Comment (LennartKoot) [inline src/RenderedHelper.ts:30]: Maybe we should look into using `os.EOL` later, instead of using Windows specific new line characters

**Review by LennartKoot — CHANGES_REQUESTED** [review]

File: src/RenderedHelper.ts (hunk 1 above)
Comment (ThumNet) [reply src/RenderedHelper.ts:13]: Would have liked that but then needed to parse the request line and account for all scenarios of HTTP request/response

File: src/RenderedHelper.ts (hunk 2 above)
Comment (ThumNet) [reply src/RenderedHelper.ts:30]: Good point, will pickup in later release!

**Review by LennartKoot — APPROVED** [review]

**Pull request merged**